gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_2_0/schema.yml

```diff
@@ -34,4 +34,4 @@ databaseChangeLog:
 
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.